src/app.module.ts

```diff
@@ -2,10 +2,29 @@ require('dotenv').config();
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { UserModule } from './users/user.module';
+import { Connection } from 'typeorm';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+import { User } from './users/user.entity';
 
 @Module({
-  imports: [],
+  imports: [
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: process.env.DB_HOST,
+      port: Number(process.env.DB_PORT),
+      username: process.env.DB_USER,
+      password: process.env.DB_PASS,
+      database: process.env.DB_NAME,
+      entities: [User],
+      synchronize: true,
+    }),
+    UserModule
+  ],
   controllers: [AppController],
   providers: [AppService],
 })
-export class AppModule {}
+export class AppModule {
+  constructor(private connection: Connection) {}
```

Comment:
@haltaction I'm not sure if it's needed. It was in example, but I'm also using `TypeOrmModule.forFeature([User])` into the UsersModule

Reply:
You could delete it/ If not sure, try to delete and watch what is happening :)